control-plane/roles/gardener/files/kube-apiserver/templates/service-kube-apiserver.yaml

```diff
@@ -15,13 +15,18 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Values.serviceName }}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: {{ .Values.kubeAPIServer.hostname }}
+  name: {{ .Values.kubeAPIServer.serviceName }}
   namespace: {{ .Release.Namespace }}
   labels:
     app: garden
     component: kube-apiserver
 spec:
-  type: ClusterIP
+  {{- if .Values.kubeAPIServer.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.kubeAPIServer.loadBalancerIP }}
+  {{- end }}
+  type: LoadBalancer
```

Comment:
Can we move the `type: LoadBalancer` into the if-condition? I would like to maintain the original behavior and do not automatically acquire an IP address if someone forgets to set this.